VTFLib/VTFFile.cpp

```diff
@@ -3556,7 +3629,7 @@ static SVTFImageConvertInfo VTFImageConvertInfo[IMAGE_FORMAT_COUNT] =
 	{},
 	{},
 	{},
-	{},
+	{	  8,  0,  0,  0,  0,  0,	-1, -1, -1, -1,	 vlTrue, vlTrue,	NULL, NULL,			IMAGE_FORMAT_BC6H},
```

Comment:
This will need to get moved to after `IMAGE_FORMAT_BC7` when you update the enum (see the later comment)

Reply:
✔